admin/src/data/modules.json

```diff
@@ -0,0 +1,65 @@
+[
```

Comment:
do we need this file? it looks like test file before we had api ready
pls remove

Reply:
not anymore